### ISSUE TITLE
Unload Equipment

### ISSUE THREAD

**Issue by d0ob** [psznm/DoubleSpeedSpidertron#2]
Hi there ! 

When I unload a full grid, by ctrl+clicking a free space, and in any vehicle grid, no only a spidertron, I end up with my inventory filled with "unknown fusion reactor spidertron double speed" and "exosqueleton spidertron double speed". I guess it is how you managed to make the mod, is there anything that can be done about that ? Could they be more than invisible, also unobtainable ? 

Thanks !

**Comment by psznm** [comment]
Thank you very much for the bug report! I can not make those items unobtainable, however I added their removal immediately when you get them, meaning you will never notice they were there. Added to 0.0.6 version.

There still may be rare issue where if you do not have at least 2 extra free spaces, you will get message some items couldn't be removed and some actual items may remain inside the grid. However I'm afraid that there is nothing I can do about that.
